test/models/financial_year_test.rb

```diff
@@ -208,4 +208,33 @@ class FinancialYearTest < ActiveSupport::TestCase
       assert f.close(nil, options), "Financial year #{f.code} should be closed"
     end
   end
+
+  test 'compute periods given a specific interval' do
+    financial_year_18_19 = FinancialYear.find_by_code('EX2018/2019')
```

Comment:
What if the given financial year doesn't exists? Shouldn't you create it first ?

Reply:
Well it comes from the fixtures (pre loaded)